src/logger.rs

```diff
@@ -4,22 +4,24 @@
 extern crate pwasm_std;
 
 use pwasm_std::{CallArgs, storage, write_u32};
+use pwasm_std::bigint::U256;
+use pwasm_std::hash::H256;
 
-fn set_key_from_addr(key: u32, val: &[u8; 20]) {
+fn set_key_from_addr(key: u32, val: &[u8]) {
 	let mut full_key = [0u8; 32];
 	let mut full_val = [0u8; 32];
 
 	write_u32(&mut full_key[0..4], key);
 	full_val[12..32].copy_from_slice(val);
```

Comment:
will panic now if val length isn't 20

Reply:
Not a big deal, it's internal function used in limited number of cases
And we handle such panics inside wasm in a controlled way